src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1,4 +1,36 @@
-// Add the required types and props
-export const TodoInfo = () => (
-  <>TodoInfo markup</>
-);
+import classNames from 'classnames';
+import React from 'react';
+import { Todo } from '../../types/Todo';
+import { UserInfo } from '../UserInfo/UserInfo';
+import './TodoInfo.scss';
+
+type Props = {
+  todo: Todo;
+};
+
+export const TodoInfo: React.FC<Props> = ({ todo }) => {
+  const {
+    title,
+    completed,
+    user,
+  } = todo;
+
+  return (
+    <article className={classNames('TodoInfo', {
+      'TodoInfo--completed': completed,
+    })}
+    >
+      <h2 className="TodoInfo__title">{title}</h2>
+      {user
+        ? (
+          <UserInfo
+            user={user}
+          />
+        )
+        : (
+          'No user found'
+        )}
```

Comment:
This way would be easier to read the code(because you have only 1 argument, not 3 or more).
```suggestion
        ? (<UserInfo user={user} /> )
        : ('No user found')}
```

Reply:
Thanks for the feedback!